meetings/2024-03-06.md

```diff
@@ -45,12 +50,93 @@ URL: https://github.com/solid/specification/discussions/618
+* HZ: Do you still want to do the poll for changing the name?
+* SC: Thinking about the name depends on the scope/mission. What do you want to get out of this poll 
+* PAC: Whether people think it is important to stick to "Solid" or not. I agree that the name depends on the mission. At this point do we feel we should stick with Solid or change.
+* HZ: I'm neutral but biased towards keeping the name because people are familiar with it. I don't know if you find a name that reflects.
+* SC: I don't have a strong opinion, I don't think it matters. In SocialWG, there was ActivityPub, AS2, LDN etc. Similarly this WG could publish a set of spec. In SocialWG some specs were overlaping. I don't think that the name has branding value to the working group. At some point it will be about sucess of the spec. I'm open to the idea to pick a different name. I think about Socially aware web, or talking about fixing the Web. I'm open to changing the name. Depends on the scope/mission/deliverables.
+* AC: I'm interested in the scope/mission. Content coming out of the WG. If it helps us move the WG along, then happy to move. Basically neutral. Leaning slightly on changing the name.
+* TBL: In principle on changing the name, as mentioned before with read-write protected/authenticated data. Even discussing in this group we have different opinions.
+* eP: Sometimes branding can create tension especially in standards. To accomplish the mission of Solid, we need something more than what this Working Group can accomplish. Solid could push a storage spec, access control spec, etc. that other things approaches can also work with.. Thre are other projects in the same perimiter and don't worry about naming them because the Solid will build on them. There is more to Solid.
+* AB: Neutral.
+* RG: The branding can be an asset or liability. Everyone knows there is something going on with Solid. Losing that could be a problem but everyone has a different idea on Solid. Compromise idea for the name would be using Solid as a different expansion. Something like "SOcial, LInked, Decentralization" or another play on Solid so keeping the branding but meaning is something more.
+* TBL: Maybe this Group can standardise different htings. Solid is an internet protocol. What bits go across the internet. Like CalDAV, IMAP - if I write a combination of different semantics. Philosophically very important to the WG because they'l be conscience of that. SC  understood ??? what ???. The Solid Protocol deliverable has a lot of value and use by different apps. The Mission defines the internet.
```

Comment:
```suggestion
* RG: Branding can be an asset or liability. Everyone knows there is something going on with Solid. Losing that could be a problem, but everyone has a different idea on Solid. Compromise idea for the name would be using Solid as a different expansion. Something like "SOcial, LInked, Decentralization" or another play on Solid so keeping the branding, but meaning something more.
* TBL: Maybe this Group can standardise different things. Solid is an internet protocol. What bits go across the internet. Like CalDAV, IMAP — if I write a combination of different semantics. Philosophically very important to the WG because they'll be conscious of that. SC understood ??? what ???. The Solid Protocol deliverable has a lot of value and use by different apps. The Mission defines the internet.
```